kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/log/AnsiLogger.java

```diff
@@ -0,0 +1,32 @@
+package org.arquillian.cube.kubernetes.impl.log;
+
+import org.arquillian.cube.kubernetes.api.Logger;
+
+import static org.fusesource.jansi.Ansi.Color.GREEN;
+import static org.fusesource.jansi.Ansi.Color.RED;
+import static org.fusesource.jansi.Ansi.Color.YELLOW;
+import static org.fusesource.jansi.Ansi.ansi;
+
+public class AnsiLogger implements Logger {
+
+    @Override
+    public void info(String msg) {
+        System.out.println(msg);
+    }
+
+    @Override
+    public void warn(String msg) {
+        System.out.println( ansi().fg(YELLOW).a(msg).reset() );
```

Comment:
Oh that is cool :) If we gonna stick to sysout maybe we can align it across the modules. The only thing is that we bring additional dep to the project, but I guess it's not really that crucial from the "client" perspective.